packages/ws-server/src/index.ts

```diff
@@ -19,10 +19,11 @@
  */
 
 import { app, httpApp } from './webSocketServer';
+import constants from "@hashgraph/json-rpc-relay/dist/lib/constants";
```

Comment:
In some placed you use `@hashgraph/json-rpc-relay/src/lib/constants` in other places you use this.
Assuming the same we should be consistent